src/NuGet.Core/NuGet.PackageManagement/NuGet.PackageManagement.csproj

```diff
@@ -1,9 +1,19 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
+  <!--
+    Import the SDK props manually _after_ we've imported the build customizations that
```

Comment:
Why not use a `Directory.Build.props` file instead?

Reply:
I wasn't sure how much rejiggering of the build that would require--my feeling was that for an initial pass, I wanted to keep the scope of changes as low as possible.